testsuite/pytests/sli2py_regressions/test_ticket_80_175_179.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# test_ticket_80_175_179.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+Regression test for ticket #80, #175 and #179.
+
+This set of test verifies that ``voltmeter`` and ``spike_recorder`` record
+identical signals from ``iaf_psc_alpha`` driven by internal DC independent
+of how simulation time is blocked and how elements are placed in script.
+Additionally, two spike generators, placed before and after the recorders,
+emit the same spike trains that are expected from the neurons for the given
+parameters.
+"""
+
+import nest
+import numpy.testing as nptest
+import pytest
+
+vm_stop = 100.0
+total_sim_time = vm_stop + 2.0
+
+
+@pytest.fixture(scope="module")
```

Comment:
If I understand the documentation here (https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) correctly, then the `setup()` method would be run only once for the entire module and we just every time look at the same data. 
```suggestion
```

Have you tried to break the test by making some changes to you are sure it does a fresh simulation for each block length?

Reply:
Oh I was under the impression that the network setup was static for all runs and only the sim block varied, the module scope definitely only runs it once for all. Can change it later once back at my PC.